services/datamanager/builtin/capture_test.go

```diff
@@ -158,7 +147,10 @@ func TestDataCaptureEnabled(t *testing.T) {
 			}()
 			err = dmsvc.Update(context.Background(), initConfig)
 			test.That(t, err, test.ShouldBeNil)
-			time.Sleep(captureTime)
+			for i := 0; i < 20; i++ {
```

Comment:
Why 20?

Reply:
Completely arbitrary. Anything >0 would work. Changed to just a single time to resolve any ambiguity

Reply:
Oof this was a lie because I forgot about buffering, and in the tabular case writing a single thing doesn't write enough data to cause anything to actually get written to disk. Changed this to 100

Reply:
Hard coding a value here seemed fragile because it's a function of the buffer size and the reading size, so if either of those were to ever change this could break. If the hard coded value is too low the test will fail with a false negative, and every increment it's too high makes the test take longer to run. To avoid this I updated this to have routines that concurrently increment the time.